rust-symcrypt/src/hash.rs

```diff
@@ -139,7 +209,7 @@ pub fn sha384(data: &[u8]) -> [u8; SHA384_RESULT_SIZE] {
 mod test {
```

Comment:
I like the way you've added tests here with the generic test methods and then specific ones for each type!

Reply:
thanks :)